src/com/hw8/concurrency/MyForkFoinPool.java

```diff
@@ -0,0 +1,46 @@
+package com.hw8.concurrency;
+
+import java.util.concurrent.ForkJoinPool;
+import java.util.concurrent.RecursiveTask;
+
+public class MyForkFoinPool {
+
+    final static int numOperations = 1000;
```

Comment:
What do you think about using modifier private here?
Also, there is some issue with the name convention. 
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
![image](https://user-images.githubusercontent.com/20267148/54765396-19529980-4c02-11e9-842e-def52d5e1b8d.png)

Reply:
+